tests/main.tf

```diff
@@ -1,20 +1,36 @@
+#terraform {
+#    backend "s3" {
+#      bucket         = "glueops-terraform-eks-backend"
+#      key            = "terraform.tfstate"
+#      region         = "us-west-2"
+#      dynamodb_table = "glueops-terraform-eks-backend"
+#  }
+#}
+
 module "captain" {
-  iam_role_to_assume = "arn:aws:iam::761182885829:role/glueops-captain"
+  #iam_role_to_assume = "arn:aws:iam::184515722743:role/OrganizationAccountAccessRole"
+  iam_role_to_assume = "arn:aws:iam::184515722743:role/captain-role"
   source             = "../"
   eks_version        = "1.26"
+  eks_cluster_name   = "captain"
   csi_driver_version = "v1.18.0-eksbuild.1"
-  vpc_cidr_block     = "10.65.0.0/26"
+  vpc_cidr_block     = "10.65.0.0/16"
```

Comment:
Does cilium require a large network? We have a customer requirement for using a /26